### PR TITLE
adding word to markdown capability

### PR DESCRIPTION
**Pull request by mkompanets**
* Adding word-to-markdown gem
* Parsing raw markdown and extracting image data
* Uploading markdown image data to s3 and replacing encoding with a url
* Adding build pack for openoffice (libraoffice) to heroku

### CODE REVIEW

**Comment by mkompanets** [comment]
markdown:
screen shot 2015-06-04 at 3 57 45 pm

preview:
screen shot 2015-06-04 at 3 57 57 pm

**Comment by dior001** [comment]
Looking

File: Gemfile [hunk 1]
```diff
@@ -42,6 +42,7 @@ gem 'omniauth-github'
 
 # Markdown.
 gem 'redcarpet'
+gem 'word-to-markdown', git: "https://github.com/benbalter/word-to-markdown"
```
Comment (dior001) [inline Gemfile]: No RubyGems.org version?

**Comment by dior001** [comment]
Great job on this @mkompanets LGTM :sake:

**Pull request merged**